File: Data Structures & Algorithms/level-order-traversal-of-binary-tree/submission-0.java
/**
 * Definition for a binary tree node.
 * public class TreeNode {
 *     int val;
 *     TreeNode left;
 *     TreeNode right;
 *     TreeNode() {}
 *     TreeNode(int val) { this.val = val; }
 *     TreeNode(int val, TreeNode left, TreeNode right) {
 *         this.val = val;
 *         this.left = left;
 *         this.right = right;
 *     }
 * }
 */

class Solution {
    public List<List<Integer>> levelOrder(TreeNode root) {
        List<List<Integer>> answer = new ArrayList<>();
        if (root == null) {
            return answer;
        }
        ArrayList<TreeNode> j = new ArrayList<>();
        j.add(root);
        while (j.size() != 0) {
            int size = j.size();
            List<Integer> l = new ArrayList<>();
            for (int i = 0; i < size; i++) {
                TreeNode node = j.remove(0);
                l.add(node.val);
                if (node.left != null) {
                    j.add(node.left);
                }
                if (node.right != null) {
                    j.add(node.right);
                }
            }
            answer.add(l);
            System.out.println(answer);
        }
        return answer;
    }
}

File: Data Structures & Algorithms/longest-palindromic-substring/submission-0.py
class Solution:
    def longestPalindrome(self, s: str) -> str:
        bestLeft = 0
        bestRight = 1
        left = 0
        while left < len(s):
            
            right = left + 1
            while right < len(s) and s[left] == s[right]:
                right += 1
            temp = right
            while left > 0 and right < len(s) and s[left - 1] == s[right]:
                left -= 1
                right += 1
            if right - left > bestRight - bestLeft:
                bestRight = right
                bestLeft = left
            left = temp
        return s[bestLeft: bestRight]